### ISSUE TITLE
"jsnext:main" in package.json isn't useful

### ISSUE THREAD

**Issue by borisirota** [dekujs/deku#407]
In order to make it useful I'll suggest:
1. remove `src` from `.npmignore` (or leave it as is - see 2)
2. babel `src` to `es` dir without the `transform-es2015-modules-commonjs` plugin so users of this library won't have to include all the plugins you use in your internal implementation and then set `"jsnext:main": "es/index.js"`. [check out redux as example](https://github.com/reactjs/redux/blob/master/package.json#L23)

**Comment by maccelerated** [comment]
Seconded.

**Comment by troch** [comment]
Also `.babelrc` to be added to `.npmignore`

**Comment by anthonyshort** [comment]
Yeah that needs to be fixed up. If anyone wants to open a quick PR with these changes I'll get it merged
